lighthouse-core/lib/asset-saver.js

```diff
@@ -198,9 +199,30 @@ function saveAssets(artifacts, audits, pathWithBasename) {
   });
 }
 
+/**
+ * Log trace(s) and associated screenshot(s) to console.
+ * @param {!Artifacts} artifacts
+ * @param {!Audits} audits
+ * @return {!Promise}
+ */
+function logAssets(artifacts, audits) {
+  return prepareAssets(artifacts, audits).then(assets => {
+    assets.map(data => {
+      log.log('devtoolslog-' + data.passName + '.json',data.devtoolsLog);
```

Comment:
template literals are more idiomatic.. and our style:  😎 
```js
log.log(`devtoolslog-${data.passName}.json`, data.devtoolsLog);
```

Reply:
This looks great, thanks!